src/components/PageAnnouncement.tsx

```diff
@@ -18,6 +17,8 @@ export interface PageAnnouncementProps
     DataAttributes {
   id?: string;
   className?: string;
+  headingLevel?: HeadingLevels;
```

Comment:
Add this to the tests. One of them probably has an "all props" description or similar.